### PR TITLE
Add script to undelete documents from couch

### PR DESCRIPTION
**Pull request by esoergel**

### CODE REVIEW

**Comment by esoergel** [comment]
@czue @dannyroberts @benrudolph

File: corehq/apps/cleanup/management/commands/undelete_docs.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+from corehq.util.couchdb_management import couch_config
+
+Results = namedtuple('Results', ['restored', 'not_found', 'not_deleted'])
+
+
+def get_deleted_doc(db, doc_id, rev):
+    res = db.get(doc_id, revs=True, rev=rev)
+    start = res['_revisions']['start']
+    ids = res['_revisions']['ids']
+    prev_revision = "{}-{}".format(start-1, ids[1])
```
Comment (dimagimon) [inline corehq/apps/cleanup/management/commands/undelete_docs.py]: E226 missing whitespace around arithmetic operator

**Comment by czue** [comment]
good stuff, thanks for putting this together so quickly!

**Comment by benrudolph** [comment]
+1!

**Pull request merged**